crates/bevy_render/src/camera/projection.rs

```diff
@@ -280,81 +282,30 @@ impl Default for PerspectiveProjection {
 ///    ..OrthographicProjection::default_2d()
 /// });
 /// ```
-#[derive(Debug, Clone, Copy, Reflect, Serialize, Deserialize)]
+#[derive(Default, Debug, Clone, Copy, Reflect, Serialize, Deserialize)]
 #[reflect(Serialize, Deserialize)]
 pub enum ScalingMode {
+    /// Match the viewport size.
+    ///
+    /// With a scale of 1, one world unit will be equal to the number of pixels in the window.
```

Comment:
```suggestion
    /// With a scale of 1, one pixel will equate to one world unit.
```
The description doesn't make sense to me, how does length unit correspond to pixels in the window which are area?
Not sure if suggestion is correct either, do lmk